apps/storybook/src/overlays/AlertDialog.stories.tsx

```diff
@@ -0,0 +1,132 @@
+import type { Meta, StoryObj } from "@storybook/react";
+
+import { AlertDialog, Button, Flex } from "@optiaxiom/react";
+import { expect, screen, userEvent, within } from "@storybook/test";
+import { useState } from "react";
+
+const meta: Meta<typeof AlertDialog> = {
+  argTypes: {
+    size: {
+      control: "select",
+      options: ["sm", "md", "lg"],
+    },
+  },
```

Comment:
we don't need to specify this manually - storybook will automatically pick the props